pom.xml

```diff
@@ -74,8 +74,6 @@
     <google.guava.version>20.0</google.guava.version>
     <oauth2.revision>151</oauth2.revision>
     <google.http.version>1.21.0</google.http.version>
-    <configuration-as-code.version>1.19</configuration-as-code.version>
```

Comment:
Why remove the configuration-as-code version prop? That seems orthogonal to the stated purpose of this PR.